generator/src/main/java/com/scottlogic/deg/generator/decisiontree/ProfileDecisionTreeFactory.java

```diff
@@ -148,14 +148,10 @@ private ConstraintNode convertConditionalConstraint(ConditionalConstraint constr
     }
```

Comment:
this entire class could maybe be updated to use builders, but that's too much work for now